wdl-lint/src/rules/blanks_between_elements.rs

```diff
@@ -0,0 +1,557 @@
+//! A lint rule for blank spacing between elements.
+
+use rowan::NodeOrToken;
+use wdl_ast::v1::InputSection;
+use wdl_ast::AstNode;
+use wdl_ast::Diagnostic;
+use wdl_ast::Diagnostics;
+use wdl_ast::Document;
+use wdl_ast::Span;
+use wdl_ast::SupportedVersion;
+use wdl_ast::SyntaxElement;
+use wdl_ast::SyntaxKind;
+use wdl_ast::SyntaxNode;
+use wdl_ast::SyntaxToken;
+use wdl_ast::ToSpan;
+use wdl_ast::VisitReason;
+use wdl_ast::Visitor;
+
+use crate::Rule;
+use crate::Tag;
+use crate::TagSet;
+
+/// The identifier for the blanks between elements rule.
+const ID: &str = "BlanksBetweenElements";
+
+/// Creates an excessive blank line diagnostic.
+fn excess_blank_line(span: Span) -> Diagnostic {
+    Diagnostic::note("extra blank line(s) found")
+        .with_rule(ID)
+        .with_highlight(span)
+        .with_fix("remove blank line(s)")
```

Comment:
```suggestion
    Diagnostic::note("extra blank line found")
        .with_rule(ID)
        .with_highlight(span)
        .with_fix("remove blank line")
```
I might be wrong here, but I think anytime there's double whitespace the `Whitespace` rule will catch it. Though maybe it makes more sense for this rule to catch it and say "no whitespace at all" instead of `Whitespace` saying "only one blank allowed" 🤔

Reply:
I opted to do it this way to highlight the entire span of problematic whitespace. The user would likely get a warning from the `Whitespace` rule for the additional blank lines, but that seems confusing if you get that warning for duplicates and then this one for the final blank line. I think it'll be clearer with this rule highlighting the full span.

Reply:
```suggestion
    Diagnostic::note("extra blank line(s) found")
        .with_rule(ID)
        .with_highlight(span)
        .with_fix("remove these blank line(s)")
```

How about this?

Reply:
`these` implies multiple, which may or not be the case.

Reply:
Maybe `the` then? It feels weird to me to have just `remove blank line(s)`, but I don't feel strongly, so move forward with whichever you like better.